component/modal/modal.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../../style.css" />
+    <link rel="stylesheet" href="modal.css" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"
+    />
+    <title>Modal</title>
+  </head>
+  <body>
+    <nav class="flex_space nav">
+      <div>
+        <h2><a href="../../index.html">Home</a></h2>
+      </div>
+      <div>
+        <span class="menulines pointer" onclick="openNav()">&#9776;</span>
+      </div>
+
+      <div class="sidenav" id="mySidenav">
+        <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">
+          &times;
+        </a>
+
+        <a href="../avatar/avatar.html">Avatar</a>
+        <a href="../alert/alert.html">Alert</a>
+        <a href="../badges/badges.html">Badges</a>
+        <a href="../button/button.html">Button</a>
+        <a href="../card/card.html">Card</a>
+        <a href="../responsive-image/responsive-image.html">Responsive-Image</a>
+        <a href="../input/input.html">Input</a>
+        <a href="./list.html">List</a>
+        <a href="../toster/toast.html">Toast</a>
+        <a href="../grid/responsive-grid.html">Grid</a>
+        <a href="../other/other.html">Other Utilities</a>
+        <a href="../navbar/navbar.html">Navbar</a>
+        <a href="./modal.html">Modal</a>
+      </div>
+    </nav>
+    <main>
+      <div class="page">
+        <h1 class="heading">Modal</h1>
+        <p class="description">
+          A modal is a dialog box/popup window that is displayed on top of the
+          current page
+        </p>
+        <p class="description">Modal create with modal-container class name</p>
+        <div>
+          <div class="flex_center">
+            <!-- here the check -->
+            <div class="modal-container">
```

Comment:
I feel that naming should be consistent here:
flex-center
modal-container

or
flex_center
modal_container